development/contribute-to-docs.md

```diff
@@ -0,0 +1,147 @@
+
+```bash
+git clone git@github.com:timescale/docs.timescale.com-content.git
+```
+
+Outside contributors will have to push their changes to a branch. You 
+will need to fork the repository, then pull from your fork, make changes to 
+a branch, then push changes to a branch on your fork. From there, you can 
+submit a pull request. See the GitHub instructions on 
+[collaborating with pull requests][github-collaborating-instructions] for 
+more details.
+
+Once cloned, you should be able to open the entire Timescale 
+documentation repository in a code editor of your choice.
+
+You can create a new branch in git as well:
+
+```bash
+git checkout -b my-branch-name
+```
+
+# Modifying a file
+
+Once you’re in your new branch you can start editing your doc as you normally would.
+
+At any time, you can see which files have been edited with the following git command:
+
+```bash
+git status
+```
+
+# Adding a new item to the page structure
+
+Often, you may want to add a new file to Timescale docs. To do so, you must 
+edit the `page-index/page-index.js` file. You should be able to glean 
+which lines to add and how to modify them based on the existing structure of 
+the file. If you need help, let us know in the 
+[Timescale community Slack][timescale-slack].
+
+# Submitting a pull request
+
+A pull request is a method of submitting contributions to an open development 
+project. It occurs when a developer asks for changes committed to an external 
+repository to be considered for inclusion in a project's main repository after 
+the peer review.
+
+Once you’ve completed all of your edits and are ready to submit a pull request, 
+you will first need to add all your edited files to your branch:
+
+```bash
+git add .
+```
+
+Now, you need to commit your changes along with a message explaining what changes 
+you’ve made to the repository. Your commit message should start with an active verb
+and longer descriptions should be in the body of your commit message.
+
+```bash
+git commit -m “Add a better description of what a feature does”
+```
+
+Then you have to upload your changes to GitHub:
+
+```bash
+git push origin my-branch-name
+```
+
+You will then need to complete the process on the 
+[Timescale Docs GitHub repository][timescale-docs-github]. Click on 
+the `New Pull Request` button and follow the instructions to create a 
+pull request using the branch you uploaded.
+
+# Making changes to a pull request
+
+Typically, you’ll be asked to make some changes to your pull request. You 
+can do so locally and re-add your changes to your branch and re-upload 
+it using the following commands:
+
+```bash
+git add .
+git commit --amend
+git push -f origin my-branch-name
+```
+
+Note that using `git amend` and force pushing are our preferred methods 
+because this is content and not code.
+
+Then go back to the [Timescale Docs GitHub repository][timescale-docs-github] 
+and request another pull request review by clicking the “cycle” button next 
```

Comment:
You should mention that this is on the specific pull request page, rather than just within the repository.
Also:
```suggestion
and request another review by clicking the “cycle” button next 
```